src/os/bluestore/BlueStore.cc

```diff
@@ -6961,6 +6978,7 @@ int BlueStore::queue_transactions(
 
   // prepare
   TransContext *txc = _txc_create(osr);
+  ++txc->osr->kv_finisher_submitting;
```

Comment:
@liewegas 
I think this line can be moved right after TransContext::STATE_IO_DONE: